app/views/profiles/show.html.erb

```diff
@@ -1 +1,5 @@
-<%= t(".supported", zipcode: @profile.zipcode) %>
+<% if @profile.supported? %>
```

Comment:
It might not be worth it, but if this could be done via decoration, you could do something like:

```ruby
<%= t(".#{@profile.supported_state}", zipcode: @profile.zipcode) %>
```

Might be heavy handed here though.